CoinDock-api/database/migrations/2022_05_17_163129_create_wallets_table.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wallets', function (Blueprint $table) {
+            $table->integerIncrements('id');
```

Comment:
use id() instead of integerIncrements('id').

Reply:
done